apps/src/gamelab/AnimationTab/NewListItem.jsx

```diff
@@ -52,10 +56,17 @@ var NewListItem = React.createClass({
   },
 
   render: function () {
+    var dynamicStyles = {
+      dottedBorder: {
+        backgroundColor: Radium.getState(this.state, 'main', ':hover')
+            ? color.lightest_purple : undefined
+      }
+    };
```

Comment:
I'm not sure I can entirely justify why I like my approach better. I think it's because I like the render method to focus as much as possible on the structure of the DOM with the styles separated. To this effect, you could even do:
```
render() {
  var isHovered = Radium.getState(this.state, 'main', ':hover');
  return (
    <div ...>
      <div style={[staticStyles.dottedBorder, isHovered && staticStyles.dottedBorderHovered}>
```

Not sure any of these approach is clearly better than another, and I'm fine with any.